src/index.html

```diff
@@ -8,7 +8,205 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <div id="header" class="header-wrapper">
+        <ul id="main-nav">
+          <li class="main-nav-item"><a href="">Home</a></li>
+        </ul>
+        <div id="nav-side-btn">
+          <button>Download my full resume</button>
+        </div>
+      </div>
+
+      <div id="banner1" class="banner1-wrapper">
```

Comment:
Instead of including a number as a part of id or class name, it's suggested to give a meaningful name, like `avatar-banner`, `carousel-banner` and so on.